game.py

```diff
@@ -190,10 +198,23 @@ def new_game(self, saved_state=None):
             if self._dots[n].type is not None:
                 self._dots[n].type = True
                 self._dots[n].set_shape(self._new_dot(self._colors[STROKE]))
-
         # Calculate the distances to the edge
         self._initialize_weights()
         self.strategy = self.strategies[self.level]
+        self.thread = threading.Thread(target = self.timer, daemon = True)
```

Comment:
Keyword args shouldn't be spaced, also why introduce threads?

Reply:
> also why introduce threads

Since there is no game loop, I had to use threads to run the stopwatch independently. I'm not sure if there is a way to avoid using it.

Reply:
I understand now, thanks.